diff --git a/libavformat/allformats.c b/libavformat/allformats.c
index d08f2f7..be0e170 100644
--- a/libavformat/allformats.c
+++ b/libavformat/allformats.c
@@ -224,7 +224,7 @@ void av_register_all(void)
     REGISTER_MUXER    (WEBM, webm);
     REGISTER_DEMUXER  (WSAUD, wsaud);
     REGISTER_DEMUXER  (WSVQA, wsvqa);
-    REGISTER_DEMUXER  (WTV, wtv);
+    REGISTER_MUXDEMUX (WTV, wtv);
     REGISTER_DEMUXER  (WV, wv);
     REGISTER_DEMUXER  (XA, xa);
     REGISTER_DEMUXER  (XWMA, xwma);
